### PR TITLE
Fix tag filter

### PR DESCRIPTION
**Pull request by hngerebara**
Things did in this PR

-  Fixed tag display for tags that should all be in uppercase
- Styled the checkboxes
N/B: The styling of the checkboxes was partially done. The is a ticket created for it in the minor edits story. 

Previous look 
Screenshot 2019-09-10 at 5 04 00 AM

Currrent look
Screenshot 2019-09-10 at 4 57 10 AM

### CODE REVIEW

File: assets/css/global.less [hunk 1]
```diff
@@ -1014,51 +1012,128 @@ body .guides .guides-section-white {
...
+            &::before {
+              background-color: @guides-primary-color;
+            }
+            &::after {
+              content: '';
+              position: absolute;
+              width: 10px;
+              height: 7px;
+              background: @guides-primary-color;
+              left: 3px;
```
Comment (Etiene) [inline assets/css/global.less]:
```suggestion
              left: 4px;
```
Just a bit of a nitpick for better alignment!

**Comment by hngerebara** [comment]
Is this good to go @Etiene?

**Comment by Etiene** [comment]
@hngerebara ah sorry, my suggested commit was for fixing something else! That alignment in the image I posted is a different one that you still need to look into, let me know if you want to pair so I can show you better

**Comment by hngerebara** [comment]
@Etiene There is a ticket to properly match checkboxes with mockup [here](https://gruntwork.atlassian.net/jira/software/projects/HSTN/boards/31?selectedIssue=HSTN-323) or should I just fix all of it here?

**Comment by Etiene** [comment]
Isnt this PR for that card? I'm confused 😅

**Comment by Etiene** [comment]
Nevermind, I just saw on Jira that this PR is a different card! I was just making sure that the vertical alignment isnt forgotten. Up to you to add it in this PR or a different one

**Pull request merged**